src/components/UI/Overlay/OverlayContent/AddressBook.tsx

```diff
@@ -43,8 +44,27 @@ export const AddressBook = styled(({ onAddressSelected, ...props }: AddressBookP
               </div>
             </div>
           </div>
-          <div className="col-auto ml-auto">
-            <CsvUploadButton />
+          <div className="col-12 col-md-auto ml-md-auto">
+            <div className="row no-gutters">
+              <div className="col-12 col-md-auto mb-2 mb-md-0">
+                <AnchorLinkButtonSolidWhiteBlue
+                  href="data:text/csv;base64,QWRkcmVzcyxJZGVudGlmaWVy"
```

Comment:
i think put in at least two examples would be helpful? maybe the addresses we use in the demo